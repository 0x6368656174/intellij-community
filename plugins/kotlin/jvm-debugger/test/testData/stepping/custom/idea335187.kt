package idea335187

fun main(args: Array<String>) {
    foo {
        fun bar() {}
        bar()

        //Breakpoint!
        println("Hello World")
    }
}

fun foo(body: () -> Unit) = body()
