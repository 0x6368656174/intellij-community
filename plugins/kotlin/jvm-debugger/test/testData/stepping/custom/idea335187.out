LineBreakpoint created at idea335187.kt:9
Run Java
Connected to the target VM
idea335187.kt:9
Disconnected from the target VM

Process finished with exit code 0
